tests/unit/lib/build_module/test_app_builder.py

```diff
@@ -20,6 +20,8 @@
 from samcli.lib.utils import osutils
 from tests.unit.lib.build_module.test_build_graph import generate_function
 
+from tests.testing_utils import IS_WINDOWS
```

Comment:
This looks unused? Was there something this was suppose to be used for?

Reply:
It was used to make TestApplicationBuilder_update_template_windows run only on Windows. Removed as it is not used anymore.